<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>한컴 고딕</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#00aa7f;&quot;&gt;네이버 이웃 추가 프로그램&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="id_label">
           <property name="font">
            <font>
             <family>한컴 고딕</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>아이디</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="id_input">
           <property name="font">
            <font>
             <family>한컴 고딕</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="pw_label">
           <property name="font">
            <font>
             <family>한컴 고딕</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>비밀번호</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="pw_input">
           <property name="font">
            <font>
             <family>한컴 고딕</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="keyword_label">
           <property name="font">
            <font>
             <family>한컴 고딕</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>키워드</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="keyword_input">
           <property name="font">
            <font>
             <family>한컴 고딕</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="max_label">
           <property name="font">
            <font>
             <family>한컴 고딕</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>최대 이웃추가수</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="max_input">
           <property name="font">
            <font>
             <family>한컴 고딕</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>150</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="message_label">
         <property name="font">
          <font>
           <family>한컴 고딕</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="start_btn">
           <property name="font">
            <font>
             <family>한컴 고딕</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>시작</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reset_btn">
           <property name="font">
            <font>
             <family>한컴 고딕</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>리셋</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="font">
            <font>
             <family>한컴 고딕</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="quit_btn">
           <property name="font">
            <font>
             <family>한컴 고딕</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>종료</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="status_label">
       <property name="font">
        <font>
         <family>한컴 고딕</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>상태 메시지</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
